### ISSUE TITLE
what is the input image

### ISSUE THREAD

**Issue by huangsusan** [lc82111/Active-Contour-Loss-pytorch#3]
Hi:
I wonder what the input image is and can I apply AC loss individually without Dice loss.
Can I apply AC loss in 3D neural network ,such as 3D UNet

best wishes

**Comment by asifehmad** [comment]
> Hi:
> I wonder what the input image is and can I apply AC loss individually without Dice loss.
> Can I apply AC loss in 3D neural network ,such as 3D UNet
> 
> best wishes

Hello, it is just a `loss function`, you can it in place of `iou`, `cross entropy` etc,
